Models/Employee.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Employee
+    {
+        [Key]
+        public int EmployeeId { get; set; }
+
+        [Required]
+        public string FirstName { get; set; }
+
+		[Required]
+        public string LastName { get; set; }
+
+		[Required]
+		public string Email { get; set; }
+
+		[Required]
+		public bool Supervisor { get; set; }
+
+		[Required]
+		public int DepartmentId { get; set; }
+		
+        public Department Department { get; set; }
```

Comment:
Ticket #6 says "A representation of the computer that the employee is currently using should be included in the employee representation". Since this PR also has the `Computer` model on it, I think we can go ahead and add that in now?

Reply:
It is now updated!